Sources/Hummingbird/Server/RequestContext.swift

```diff
@@ -92,7 +92,7 @@ extension RequestContext {
     public var parameters: Parameters { coreContext.parameters }
     /// Request ID, extracted from Logger
     @inlinable
-    public var id: String { self.logger[metadataKey: "hb_id"]!.description }
+    public var id: String { self.logger[metadataKey: "hb.id"]!.description }
```

Comment:
```suggestion
    public var id: String { self.logger[metadataKey: "http.request.id"]!.description }
```

Reply:
Used an `hb` prefix instead of http prefix as `http.request.id` is not in the Otel spec and we don't own the http namespace